catalog/weather/forecast.js

```diff
@@ -21,30 +21,28 @@ function main(params) {
     var language = params.language || 'en-US';
     var units = params.units || 'm';
     var timePeriod = params.timePeriod || '10day';
-    var timeURL ='/forecast/daily/10day';
+    var url = 'https://twcservice.mybluemix.net/api/weather/v1/geocode/' + lat + '/' + lon;
```

Comment:
Never mind I see the test now

> https://new-console.ng.bluemix.net/docs/services/InsightsWeather/weather_rest_apis.html

> This service is deprecated: All instances of this service are deprecated. The revised service is available in Bluemix as Weather Company Data for IBM Bluemix.
Current apps that use the Insights for Weather APIs will continue to work without any modifications for 90 days after the Weather Company data service is released on July 1 2016. To take advantage of the newly added APIs and the improved pricing model, it is in your best interest to migrate to one of the new plans.